desktop-app/src/renderer/components/DeviceManager/index.tsx

```diff
@@ -11,13 +11,15 @@ import {
 import { APP_VIEWS, setAppView } from 'renderer/store/features/ui';
```

Comment:
thanks for the PR, please follow the following structure: 

- Make a file named DeviceManager.tsx for this file
- In the index.tsx file just do `export { DeviceManager } from './DeviceManager` -> this acts as a export hub for anything you need to export
- Create separate file components for the lines of code I am commenting below